enhancements/update/versioning-cincinnati-api.md

```diff
@@ -0,0 +1,116 @@
+---
+title: versioning-cincinnati-api
+authors:
+- "@pratikmahajan"
+  reviewers:
+- @openshift/openshift-team-cincinnati-maintainers
+- @openshift/openshift-team-cincinnati
+- @openshift/team-cincinnati-operator
+  approvers:
+- @crawford
+- @LalatenduMohanty
+- @sdodson
+  creation-date: 2021-08-20
+  last-updated: 2021-08-20
+  status: implementable
+  see-also:
+-
+replaces: []
+superseded-by: []
+---
+
+# Versioning cincinnati api and json schema
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+
+## Summary
+As of today, we declare the schema version in Cincinnati Graph URI. We currently, do not embed the version information
+in json schema. The enhancement will embed the version information in the json schema. This has the benefit of being 
```

Comment:
nit: [trailing spaces][1] here and elsewhere in the file.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_enhancements/870/pull-ci-openshift-enhancements-master-markdownlint/1428436850845945856#1:build-log.txt%3A18

Reply:
fixed